release-please-config.json

```diff
@@ -51,8 +51,6 @@
   ],
   "packages": {
     ".": {
```

Comment:
Not an issue, but is there a reason to remove this?

Reply:
- According to the [documentation](https://github.com/googleapis/release-please/blob/main/docs/manifest-releaser.md) the `package-name` is ignored by node releases, so I just removed it because it's not used
- According to the [documentation](https://github.com/googleapis/release-please/blob/08331fa555ed482f675fb3e9dd5b0a5296e200a3/schemas/config.json#L77C10-L77C34) `include-component-in-tag` is `true` by default.

Reply:
Got it, thanks!